docs/running.rst

```diff
@@ -25,43 +25,45 @@ temporarily for a particular "environment" or directory. We use
 `virtualenvwrapper`_. Assuming you're in the repo root, do the following to
```

Comment:
It might be worth keeping in because the command `workon metadeploy` is mentioned later, and that requires `virtualenvwrapper`. I can update the instructions to just rely on plain `python -m venv` though, but I think that convenience is nice 🤷

Reply:
Fixed in 7f9c7f3674d5e475d07804db947e961cf6a80628